src/styles/index.scss

```diff
@@ -1,3 +1,117 @@
+@import './utils/variable';
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+/* #region card */
+
+.block {
+  box-sizing: border-box;
+  display: flex;
+  width: 200px;
+  background-color: #fff;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.image {
+  display: flex;
+  background-image: url(/src/images/imac.jpeg);
+  background-size: cover;
+  background-position: center;
+  height: 134px;
+  width: 160px;
+  margin: 32px auto 0;
+}
+
+.product-name {
+  display: flex;
+  font-weight: 500;
+  padding-inline: $padding;
+  color: $main-text-color;
+  margin-top: 40px;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.product-code {
+  margin-top: 4px;
+  display: flex;
+  font-weight: 400;
+  color: $additional-text-color;
+  font-size: 10px;
+  padding-left: $padding;
+  line-height: 14px;
+}
+
+.review {
+  display: flex;
+  padding-inline: $padding;
+  justify-content: space-between;
+  margin-top: 16px;
+  align-items: flex-end;
+
+  @import '/src/styles/blocks/stars';
```

Comment:
The import statement for '/src/styles/blocks/stars' should not be inside a CSS rule. Consider moving it to the top of the file or outside any CSS rules.